manifest/com.frozenreflex/Thundagun/info.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "Thundagun",
+	"id": "com.frozenreflex.Thundagun",
+	"description": "A lightning fast performance mod for Resonite. It improves performance by decoupling the FrooxEngine and Unity loops by running FrooxEngine and Unity on separate threads.",
+	"category": "Optimization",
+	"sourceLocation": "https://github.com/Frozenreflex/Thundagun",
+	"additionalAuthors": [
+		"989onan",
+		"DoubleStyx",
+		"Nytra",
+		"Merith-TK"
+	],
+	"versions": {
+		"1.1.1": {
+			"releaseUrl": "https://github.com/Frozenreflex/Thundagun/releases/tag/1.1.1",
+			"artifacts": [{
+				"url": "https://github.com/Frozenreflex/Thundagun/releases/download/1.1.1/Thundagun.dll",
```

Comment:
Non-blocking: Seems there is also `[assembly: AssemblyInformationalVersion("1.0.0+627300593965b723ad0e14a9c0e5d5e7d713819a")]` being added in your build. Not important here but noticed it.